### ISSUE TITLE
DB/Conditions: Kharan Mighthammer

### ISSUE THREAD

**Issue by ZenoX92** [TrinityCore/TrinityCore#22181]
**Description:**

* Fix gossip text conditions for Kharan Mighthammer
(His gossip text depends on whether the player is Alliance or Horde.)

**Branch(es):**

3.3.5

**TC rev. hash/commit:** 

TrinityCore rev. 1cc3d33 2018-02-19 20:21:57 +0100 (3.3.5 branch) (Win64, RelWithDebInfo, Static)

**TDB version:**  335.64

**Operating system:** Windows 7

**Suggested Fix:**
```sql
DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=14 AND `SourceGroup`=1822;
INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
(14,1822,2473,0,0,6,0,67,0,0,0,0,0,"","Show gossip text 2473 if player is Horde"),
(14,1822,2474,0,0,6,0,469,0,0,0,0,0,"","Show gossip text 2474 if player is Alliance");

**Comment by illfated** [comment]
Thank you. 👍 This part escaped my attention back when I moved the core script to SAI.
(commit https://github.com/TrinityCore/TrinityCore/commit/2125c126097812fd6682db8edfd543c0d5583abf)

**Comment by ccrs** [comment]
That work you did those days was quite a hard and tedious one @illfated, did you actually finish with all of them?

**Comment by illfated** [comment]
Not quite, I did collaborate with @velinath for the task of cleaning up hardcoded text in C++ scripts, whereas the overarching goal was to move C++ scripts to SAI for as many non-boss NPCs as possible.
See also https://github.com/velinath/TrinityCore/issues?utf8=✓&q=is%3Aissue for the details.

During the last couple of years, I came to the realization that I currently do not have as much free time as I would like to spend on scripts and converting them. 😞 I enjoy helping out if and when I can, though.
